engine/runtime/src/main/java/org/enso/interpreter/node/MethodRootNode.java

```diff
@@ -83,6 +85,27 @@ public static MethodRootNode build(
     return new MethodRootNode(language, localScope, moduleScope, body, section, type, methodName);
   }
 
+  public static MethodRootNode buildOperator(
+      EnsoLanguage language,
+      LocalScope localScope,
+      ModuleScope moduleScope,
+      Supplier<ExpressionNode> readLeft,
+      Supplier<ExpressionNode> readRight,
+      Supplier<ExpressionNode> body,
+      SourceSection section,
+      Type type,
+      String methodName) {
+    Supplier<ExpressionNode> supplyWholeBody =
+        () -> {
+          var readLeftNode = readLeft.get();
+          var readRightNode = readRight.get();
+          var exprNode = body.get();
+          var operatorNode = new BinaryOperatorNode(readLeftNode, readRightNode, exprNode);
+          return BlockNode.build(NO_STATEMENTS, operatorNode);
```

Comment:
I assume there is a reason, but I'd like to understand why we need to return a block node 
with no statements instead of just the built operator?

```suggestion
          return operatorNode;
```